glpk/bld.bat

```diff
@@ -0,0 +1,14 @@
+cd w%ARCH%
+copy config_VC config.h
+
+nmake /f Makefile_VC
+
+mkdir %PREFIX%\bin

```

Comment:
Thanks for reporting this. If you have time, please submit a PR so that you
get credit for the fix.

On Mon, Feb 15, 2016, 02:24 Juan Luis Cano Rodríguez <
notifications@github.com> wrote:

> In glpk/bld.bat
> <https://github.com/conda/conda-recipes/pull/481#discussion_r52870227>:
>
> > @@ -0,0 +1,14 @@
> > +cd w%ARCH%
> > +copy config_VC config.h
> > +
> > +nmake /f Makefile_VC
> > +
> > +mkdir %PREFIX%\bin
>
> Original issue: CAChemE/pyomo-recipes#6
> <https://github.com/CAChemE/pyomo-recipes/issues/6>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-recipes/pull/481/files#r52870227>.
>